### DISCUSSION TITLE
Can't load pdf - freetype issue?

### DISCUSSION THREAD

**Discussion by H2SO4-X** [Audiveris/audiveris#759] [Q&A]
Hello,

I tried installing Audiveris on my Mac (Java 17.0.12) but I can't load vectorized pdf files in the software. Even the data/examples/Dichterliebe01.pdf file is raising an error.

The first time I open a pdf file the error is: 
- `Error in performing LOAD, BINARY java.util.concurrent.ExecutionException: java.lang.ExceptionInInitializerError`

Opening the same file (or another) again gives then:
- `Error in performing LOAD, BINARY java.util.concurrent.ExecutionException: java.lang.NoClassDefFoundError: Could not initialize class de.intarsys.cwt.freetype.Freetype`

I have run `brew install freetype` so I thought that would take care of the Freetype dependencies? Do you need to specify where the Freetype is located?

And I don't know if it has anything to do with this issue, but Audiveris doesn't find the OCR engine, even though I ran `brew install tesseract`. I am obviously lacking experience in building from source :)

Help would be very much appreciated!

**Comment by hbitteur** [comment]
@H2SO4-X 

I don't have access to a Mac, so take my advice with a grain of salt.

On a Mac, we don't have an Audiveris installer yet. You have to clone the project locally and build it from source yourself.
I would recommend fetching and compiling from the "development" branch, instead of the default "master" branch which is several months older.
Pay attention to the Java version: you'll need Java 21 for the development branch.

Regarding Freetype, I can't help you. Sorry.

Regarding Tesseract, don't  install it, since it is used by Audiveris as a _linked library_ and not a stand-alone executable. See detailts in [this Handbook section](https://audiveris.github.io/audiveris/_pages/install/languages/).
